regen/embed.pl

```diff
@@ -128,24 +128,34 @@ sub generate_proto_h {
         die_at_end "$plain_func: u flag only usable with m" if $flags =~ /u/
                                                             && $flags !~ /m/;
 
+        my %static_flags;
+        while ($flags=~/([SsIi])/g) {
```

Comment:
Replacing the entire loop with a slice would be simpler and cleaner:
```perl
@static_flags{ $flags =~/[SsIi]/g } = ();
```

I'm not sure what version of Perl introduced hash slices (as opposed to hash key/value slices) but I note that despite this file claiming to support `5.004` it contains at least one use of defined-or `//=` so clearly 5.10 is at least acceptable for this file in this day and age!

Reply:
Thanks for the suggestion. After playing with this a bit I went a slightly different route that allows me to minimize having to call keys on %static_flags so many times.